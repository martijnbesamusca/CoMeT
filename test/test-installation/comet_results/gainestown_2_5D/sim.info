{'cmdline': ['/home/martijn/Documents/uva/thesis/CoMeT/run-sniper',
             '-v',
             '-s',
             'memTherm_core',
             '-n',
             '4',
             '-c',
             'gainestown_2_5D',
             '--',
             '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test'],
 'git_revision': '70a94ea0a46a09195fc7cc97e7411ec90733f6a3',
 'host': '114c1c70b953',
 'pin_version': 'unknown',
 'rusage': (26.623968,
            1.44718,
            71588,
            0,
            0,
            0,
            38255,
            14,
            0,
            9104,
            3480,
            0,
            0,
            0,
            247802,
            285),
 'snipercmd': ['bash',
               '-c',
               '/home/martijn/Documents/uva/thesis/CoMeT/lib/sniper -c /home/martijn/Documents/uva/thesis/CoMeT/config/base.cfg --general/total_cores=4 --general/output_dir=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation --config=/home/martijn/Documents/uva/thesis/CoMeT/config/nehalem.cfg --config=/home/martijn/Documents/uva/thesis/CoMeT/config/gainestown_2_5D.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/sim.scripts.py -g --hooks/script0args= -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpsYx2V2/run_benchmarks'],
 't_elapsed': 31.269760847091675,
 't_start': 1653910825.408533,
 'tracecmds': [['/home/martijn/Documents/uva/thesis/CoMeT/record-trace',
                '-o',
                '/tmp/tmpsYx2V2/run_benchmarks',
                '-v',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                '/home/martijn/Documents/uva/thesis/CoMeT/test/test-installation/test']],
 'user': None,
 'vmem': 453079040L}
