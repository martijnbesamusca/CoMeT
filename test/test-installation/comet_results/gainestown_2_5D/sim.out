                                     | Core 0     | Core 1     | Core 2     | Core 3    
  Instructions                       |   21246393 |          0 |          0 |          0
  Cycles                             |    9101160 |    9101160 |    9101160 |    9101160
  IPC                                |       2.33 |       0.00 |       0.00 |       0.00
  Time (ns)                          |    2528100 |    2528100 |    2528100 |    2528100
  Idle time (ns)                     |        -13 |    2528100 |    2528100 |    2528100
  Idle time (%)                      |      -0.0% |     100.0% |     100.0% |     100.0%
Branch predictor stats               |            |            |            |           
  num correct                        |    2597546 |          0 |          0 |          0
  num incorrect                      |      26959 |          0 |          0 |          0
  misprediction rate                 |      1.03% |      0.00% |      0.00% |      0.00%
  mpki                               |       1.27 |       0.00 |       0.00 |       0.00
TLB Summary                          |            |            |            |           
  I-TLB                              |            |            |            |           
    num accesses                     |    2890060 |          0 |          0 |          0
    num misses                       |         46 |          0 |          0 |          0
    miss rate                        |      0.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.00 |       0.00 |       0.00 |       0.00
  D-TLB                              |            |            |            |           
    num accesses                     |    7876047 |          0 |          0 |          0
    num misses                       |        903 |          0 |          0 |          0
    miss rate                        |      0.01% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.04 |       0.00 |       0.00 |       0.00
  L2 TLB                             |            |            |            |           
    num accesses                     |        949 |          0 |          0 |          0
    num misses                       |        465 |          0 |          0 |          0
    miss rate                        |     49.00% |      0.00% |      0.00% |      0.00%
    mpki                             |       0.02 |       0.00 |       0.00 |       0.00
Cache Summary                        |            |            |            |           
  Cache L1-I                         |            |            |            |           
    num cache accesses               |    2890060 |          0 |          0 |          0
    num cache misses                 |        936 |          0 |          0 |          0
    miss rate                        |      0.03% |       inf% |       inf% |       inf%
    mpki                             |       0.04 |        inf |        inf |        inf
  Cache L1-D                         |            |            |            |           
    num cache accesses               |    7876047 |          0 |          0 |          0
    num cache misses                 |      49535 |          0 |          0 |          0
    miss rate                        |      0.63% |       inf% |       inf% |       inf%
    mpki                             |       2.33 |        inf |        inf |        inf
  Cache L2                           |            |            |            |           
    num cache accesses               |      50956 |          0 |          0 |          0
    num cache misses                 |      48936 |          0 |          0 |          0
    miss rate                        |     96.04% |       inf% |       inf% |       inf%
    mpki                             |       2.30 |        inf |        inf |        inf
DRAM summary                         |            |            |            |           
  num dram accesses                  |      20289 |      20291 |      20313 |      20753
  average dram access latency (ns)   |      28.53 |      28.56 |      28.67 |      34.49
  average dram queueing delay        |       0.10 |       0.13 |       0.24 |       6.07
  average dram bandwidth utilization |      6.00% |      6.00% |      6.00% |      6.00%
Coherency Traffic                    |            |            |            |           
  num loads from dram                |      16311 |          0 |          0 |          0
  num loads from dram cache          |          0 |          0 |          0 |          0
  num loads from remote cache        |          0 |          0 |          0 |          0
